--- node_modules/next/dist/server/document.js	2018-10-02 17:27:37.000000000 +1000
+++ node_modules/next/dist/server/document.js	2018-10-03 14:40:37.000000000 +1000
@@ -27,6 +27,8 @@
 
 var _htmlescape = _interopRequireDefault(require("htmlescape"));
 
+var __jsesc = _interopRequireDefault(require("jsesc"));
+
 var _server = _interopRequireDefault(require("styled-jsx/server"));
 
 /* eslint-disable */
@@ -352,7 +354,7 @@
       var __NEXT_DATA__ = documentProps.__NEXT_DATA__;
       var page = __NEXT_DATA__.page,
           pathname = __NEXT_DATA__.pathname;
-      return "__NEXT_DATA__ = ".concat((0, _htmlescape.default)(__NEXT_DATA__), ";__NEXT_LOADED_PAGES__=[];__NEXT_REGISTER_PAGE=function(r,f){__NEXT_LOADED_PAGES__.push([r, f])}").concat(page === '/_error' ? ";__NEXT_REGISTER_PAGE(".concat((0, _htmlescape.default)(pathname), ",function(){var e = new Error('Page does not exist: ").concat((0, _htmlescape.default)(pathname), "');e.statusCode=404;return {error:e}})") : '');
+      return "__NEXT_DATA__ = ".concat((0, _htmlescape.default)(__NEXT_DATA__), ";__NEXT_LOADED_PAGES__=[];__NEXT_REGISTER_PAGE=function(r,f){__NEXT_LOADED_PAGES__.push([r, f])}").concat(page === '/_error' ? ";__NEXT_REGISTER_PAGE(".concat((0, _htmlescape.default)(pathname), ",function(){var e = new Error('Page does not exist: ").concat((0, __jsesc.default)(pathname), "');e.statusCode=404;return {error:e}})") : '');
     }
   }]);
   return NextScript;
